lib/chef/application.rb

```diff
@@ -102,6 +102,12 @@ def load_config_file
       Chef::Config.merge!(config)
     end
 
+    def set_specific_recipes
+      Chef::Config[:specific_recipes] =
+        cli_arguments.map { |file| File.expand_path(file) } if
+        cli_arguments.respond_to?(:empty) && !cli_arguments.empty?
```

Comment:
Nil doesn't respond to `#empty?`, which short circuits the expression, so is quiet when no additional ARGV are supplied.